README.md

```diff
@@ -161,23 +161,19 @@ import (
 	"log/slog"
 
 	fdk "github.com/CrowdStrike/foundry-fn-go"
-	"github.com/CrowdStrike/gofalcon/falcon"
-	"github.com/CrowdStrike/gofalcon/falcon/client"
+	"github.com/crowdstrike/gofalcon/falcon"
```

Comment:
this is very interesting, why does it need to be lowercase for gofalcon but not foundry-fn-go? Its the same github org yah?

Reply:
The difference is from the go.mod - it's lowercase in gofalcon but capital CS in foundry-fn-go.

Reply:
super weird, curious why that's the case 🤔

Reply:
AFAIK all lower is the convention

- "By convention, packages are given lower case, single-word names", https://go.dev/doc/effective_go#names
- "Good package names are short and clear. They are lower case, with no under_scores or mixedCaps", https://go.dev/blog/package-names
- And just looking at e.g. https://github.com/CrowdStrike/gofalcon/blob/main/go.mod

Reply:
I know we've had users say they have to use the mixed case for `go get`, or it fails, which is super annoying. Ideally it'd be all lowercase. I'll have to figure out what's going on there.